app/views/admin/loan_questions/_form.slim

```diff
@@ -16,9 +16,21 @@
 
     = f.input :data_type, as: :select, label: t('loan_questions.data_type'),
       collection: @data_types
-      
+
     = f.input :has_embeddable_media, label: t('loan_questions.has_embeddable_media')
 
+    label Required for which loan types?
+
+    - if @loan_question.parent
+      = f.collection_radio_buttons :override_associations,
+        [[false, 'Inherits from parent'], [true, 'Overrides parent']], :first, :last,
+        item_wrapper_class: 'radio'
+
+    - hideLoanTypes = @loan_question.parent && !@loan_question.override_associations ? 'hidden': ''
+    .loan-types-container class="#{hideLoanTypes}"
+      = f.input_field :loan_type_ids, collection: @loan_type_options, multiple: true,
+        class: "loan-types"
```

Comment:
I spoke with Fuzzy about this. There are translations in the database, which can be accessed in rails console with `Loan.loan_type_options`. I am using that dataset with the translations in this spot.